### ISSUE TITLE
New CLI version

### ISSUE THREAD

**Issue by mikeumus** [stevefarwell/angular2-snapsvg-demo#1]
Loading snapsvg into an angular app right now i a pain, it'd be nice to see it done in an exmple project like this. Thanks for the previous version.

**Comment by stevefarwell** [comment]
are you using angular cli? if so what version?

**Comment by mikeumus** [comment]
rc2, I got it working with `snapsvg-cjs` like 

```
import Snap from 'snapsvg-cjs`
import mina from `snagsvg-cjs`
```

it can't do them together like `import { Snap, mina } from 'snapsvg-cjs'`

**Comment by stevefarwell** [comment]
thats great, I was just sitting down to take a look at updating this. I'm not using svg/snap currently 
have you set up repo?

**Comment by mikeumus** [comment]
Yeah but it's a private project at work I can't share, unfortunately. You can see the `package.json` and `.angular-cli.json` here in a Stack Overflow question of mine: http://stackoverflow.com/q/42846582/1762493

**Comment by EasonPai** [comment]
I have similar problem which is really pain in the ass!

Today I am trying to update angular cli from "1.0.0-beta.18" to "1.0.0"
The old way you provided is no longer working.

When I run my app, I get this error right away:
`Uncaught TypeError: Cannot read property 'on' of undefined`

Totally have no idea what to do with this.

**Comment by EasonPai** [comment]
@mikeumus Do you have a solution now?

**Comment by EasonPai** [comment]
For your reference, I have forked this project and re-built project with "@angular/cli": "1.0.0"
[check the repo here](https://github.com/EasonPai/angular2-snapsvg-demo/tree/ngcli-v1)

**Comment by EasonPai** [comment]
oh no! it works by using "snapsvg-cjs": "0.0.4" & import "snapsvg-cjs";

**Comment by stevefarwell** [comment]
thats great, I'll check it out and update

**Comment by stevefarwell** [comment]
updated for cli v1 and angular v4

**Comment by mikeumus** [comment]
Very cool. Thanks @stevefarwell 😄

**Comment by EasonPai** [comment]
Hi~ thanks for the update,  but I have encountered a problem,
when I add a line `public canvas: Snap.Element;` to app.component.ts

 I get error when I try to ng start/build:

> ERROR in [...] angular2-snapsvg-demo-master/src/app/app.component.ts (14,17): Cannot find namespace 'Snap'.

How to get it to work? Thanks~

**Comment by EasonPai** [comment]
I managed to make it work

- use `import * as Snap from "snapsvg-cjs";` instead, see [app.component.ts](https://github.com/EasonPai/angular2-snapsvg-demo/blob/ngcli-v1/src/app/app.component.ts)  


- and add `"@types/snapsvg": "^0.4.26"` to [package.json](https://github.com/EasonPai/angular2-snapsvg-demo/blob/ngcli-v1/package.json)

work like a charm for me.
